test/promote_file.md

```diff
@@ -0,0 +1,32 @@
+Mdx can also understand ocaml code blocks:
+
+```ocaml file=toto.ml,part=toto
```

Comment:
that should be `file=promote.ml` right?

Also you need to add the relevant runes in the `dune` file.